cypress/e2e/workflow/login.cy.js

```diff
@@ -0,0 +1,24 @@
+describe("login test", () => {
+  beforeEach(() => {
+    cy.visit("/");
+    cy.wait(400);
+    cy.get("#registerModal button")
+      .contains("Login")
+      .should("be.visible")
+      .wait(500)
+      .click()
+      .wait(500);
+  });
+
+  it("testing login function, your able to login with valid email and password", () => {
+    cy.get("#loginEmail").type("jonashope@stud.noroff.no");
+    cy.get("#loginPassword").type("123123123");
+    cy.get("#loginForm button").contains("Login").wait(400).click();
```

Comment:
This covers everything up to login, but nothing after login. You could check localstorage, or the URL to see that this worked.